manager/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("@io_bazel_rules_docker//docker/util:run.bzl", "container_run_and_commit")
+load("@io_bazel_rules_docker//container:container.bzl", "container_image", "container_push")
+
+container_run_and_commit(
+    name = "server-image",
+    image = "@ubuntu-20.04//image",
```

Comment:
why ubuntu-20.04 for the license manager? No need to pull in a distro at all for a pure go binary docker container.
Use distroless instead: https://github.com/GoogleContainerTools/distroless/blob/main/examples/go/BUILD

It's both easier than what you are doing, creates much smaller images, with less security issues, which don't need to be updated (eg, if we base on ubuntu-20.04, eventually we'll need to udpate).